pkg/app/src/provider/app/modules/app.ts

```diff
@@ -0,0 +1,53 @@
+import { AppAction, AppActionTypes } from 'src/@types/app'
+import type { Dispatch } from 'react'
+
+// TODO: get root config through .env
+const HOST = 'api/'
+const ENV = 'dev'
+
+async function loadAppDat(resource: string, env: string) {
```

Comment:
please add an 'a', but will be replaced with a graphql query at some point

Reply:
🥇